### ISSUE TITLE
[STORM-504] Class Used by `repl` Command Deprecated

### ISSUE THREAD

**Issue by jira-importer** [apache/storm#4346]
No Description

---
<details><summary><i>Originally reported by <a title="Homer Strong" href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=homerstrong">homerstrong</a>, imported from: <a href="https://issues.apache.org/jira/browse/STORM-504" target="_blank">Class Used by `repl` Command Deprecated</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="Homer Strong" href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=homerstrong">homerstrong</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2014-10-01T23:44:34+01:00
<li><b>imported</b>: 2025-01-24
</ul></i>
</details>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>GitHub user strongh opened a pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/263" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/263</a></p>

<p>    Change clojure class called by `storm repl`</p>

<p>    Currently using `storm repl` there's a message saying that the `clojure.lang.Repl` class is deprecated.</p>

<p>    ```<br/>
    WARNING: clojure.lang.Repl is deprecated.<br/>
    Instead, use clojure.main like this:<br/>
    java -cp clojure.jar clojure.main -i init.clj -r args...<br/>
    Clojure 1.5.1<br/>
    ```</p>

<p>    This change merely take the suggestion of the output, and updates the class to `clojure.main`.</p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-504" class="external-link" rel="nofollow">https://issues.apache.org/jira/browse/STORM-504</a></p>

<p>You can merge this pull request into a Git repository by running:</p>

<p>    $ git pull <a href="https://github.com/strongh/incubator-storm" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/strongh/incubator-storm</a> storm-repl-class</p>

<p>Alternatively you can review and apply these changes as the patch at:</p>

<p>    <a href="https://github.com/apache/storm/pull/263.patch" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/263.patch</a></p>

<p>To close this pull request, make a commit to your master/trunk branch<br/>
with (at least) the following in the commit message:</p>

<p>    This closes #263</p>

<hr />
<p>commit 0b457f22865e4ab3be5b55f17271528b2c461c70<br/>
Author: Homer Strong <homer.strong@gmail.com><br/>
Date:   2014-09-22T22:53:12Z</p>

<p>    change clojure class called by `storm repl`</p>

<hr />

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user asfgit closed the pull request at:</p>

<p>    <a href="https://github.com/apache/storm/pull/263" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/263</a></p>
